src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -30,13 +30,42 @@
                 token)]
     result))
 
+(defn- filter-networks
+  [chain-ids network-details]
+  (filter (fn [{:keys [chain-id]}]
+            (contains? chain-ids chain-id))
+          network-details))
+
+(defn- assoc-network-preference-names
+  [network-details account testnet?]
+  (let [{:keys [prod-preferred-chain-ids
+                test-preferred-chain-ids]} account
+        current-chain-ids                  (if testnet?
+                                             test-preferred-chain-ids
+                                             prod-preferred-chain-ids)
+        network-preference-names           (->> network-details
+                                                (filter-networks current-chain-ids)
+                                                (map :network-name)
+                                                (set))]
+    (assoc account :network-preference-names network-preference-names)))
+
+(re-frame/reg-sub
+ :wallet/current-viewing-account-address
+ :<- [:wallet]
+ :-> :current-viewing-account-address)
+
 (re-frame/reg-sub
  :wallet/accounts
  :<- [:wallet]
- :-> #(->> %
-           :accounts
-           vals
-           (sort-by :position)))
+ :<- [:wallet/network-details]
+ (fn [[wallet network-details]]
+   ;; will get in follow up
+   (let [testnet? false]
```

Comment:
Why do we have a constant value  for `testnet?`

Reply:
This value will relevant after implementation -  https://github.com/status-im/status-mobile/issues/17826